packages/frontend/core/src/components/root-app-sidebar/index.tsx

```diff
@@ -93,27 +93,35 @@ export const RootAppSidebar = (): ReactElement => {
   const allPageActive = currentPath === '/all';
 
   const pageHelper = usePageHelper(currentWorkspace.docCollection);
-  const createPage = useCallback(() => {
-    return pageHelper.createPage();
-  }, [pageHelper]);
 
-  const onClickNewPage = useAsyncCallback(async () => {
-    const page = createPage();
-    page.load();
-    openPage(currentWorkspaceId, page.id);
-    mixpanel.track('DocCreated', {
-      segment: 'navigation panel',
-      module: 'bottom button',
-      control: 'new doc button',
-      category: 'page',
-      type: 'doc',
-    });
-  }, [createPage, currentWorkspaceId, openPage]);
+  const onClickNewPage = useAsyncCallback(
+    async (e?: MouseEvent) => {
+      const page = pageHelper.createPage('page', false);
+      page.load();
+      mixpanel.track('DocCreated', {
+        segment: 'navigation panel',
+        module: 'bottom button',
+        control: 'new doc button',
+        category: 'page',
+        type: 'doc',
+      });
+      if (e?.ctrlKey || e?.metaKey) {
+        if (environment.isDesktop) {
+          await workbench.newTab('/' + page.id, { show: true });
```

Comment:
It would be great if we could use the same API in the browser and desktop. We can add a `NewTabProvider` parameter to the workbench, connect different implementations through the DI container, use `popupWindow` in the browser and use `newTab` in the desktop.